samples/hubspot/program.py

```diff
@@ -0,0 +1,38 @@
+"""Demonstration of AutoKitteh's HubSpot integration.
+
+This script showcases two basic operations with the HubSpot API:
+creating a new contact and listing all deals.
+"""
+
+from autokitteh.hubspot import hubspot_client
+from hubspot.crm.contacts import SimplePublicObjectInputForCreate
+from hubspot.crm.contacts.exceptions import ApiException
+
+
+hubspot = hubspot_client("hubspot_conn")
+
+
+def create_contact(event):
+    contact_properties = {
```

Comment:
At least check if the event contains any of these parameters - if not, use these default values.